docs/framework/wcf/samples/ws-reliable-session.md

```diff
@@ -69,7 +69,7 @@ Press <ENTER> to terminate client.
   
 1. Install ASP.NET 4.0 using the following command.  
```

Comment:
This section needs to be re-written.  Starting from .NET 4.0, ASP.NET could be enabled by either enabling the windows component or running dism command.  I would suggest to simplify this to "Enable ASP.NET 4.x ...".  Is there a documentation on how to enable ASP.NET through windows component UI?

Reply:
@Rick-Anderson do you know?

Reply:
[To install IIS and ASP.NET modules on Windows Server 2012 using the UI](https://docs.microsoft.com/en-us/iis/application-frameworks/scenario-build-an-aspnet-website-on-iis/configuring-step-1-install-iis-and-asp-net-modules#to-install-iis-and-aspnet-modules-on-windows-server-2012-using-the-ui)